### ISSUE TITLE
Building properties should give an error or warning when dataframes don't merge as intended

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#1182]
In `building_properties.get_properties_technical_systems()` when dataframes are merged like so:

```
df_emission_heating = prop_HVAC.merge(prop_emission_heating, left_on='type_hs', right_on='code')
df_emission_cooling = prop_HVAC.merge(prop_emission_cooling, left_on='type_cs', right_on='code')
df_emission_control_heating_and_cooling = prop_HVAC.merge(prop_emission_control_heating_and_cooling, left_on='type_ctrl', right_on='code')
df_emission_dhw = prop_HVAC.merge(prop_emission_dhw, left_on='type_dhw', right_on='code')
df_ventilation_system_and_control = prop_HVAC.merge(prop_ventilation_system_and_control, left_on='type_vent', right_on='code')
```

buildings just get deleted from the dataframe if the corresponding `code` in the `cea.database....` does not exist.
This should at least throw a warning and merge a default system, e.g. `T1` or break down completely.

see also #1180

**Comment by gabriel-happle** [comment]
@daren-thomas do you already know the solution to that? Maybe `pandas` has some inbuilt warnings for unsuccessful merges...
Or should I take the issue?

**Comment by daren-thomas** [comment]
https://pandas.pydata.org/pandas-docs/stable/merging.html

basically, you can use the `how` argument to specify how the merging is performed. According to [the documentation](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.merge.html) the default is `inner`, which only joins those that match up.

What you'd need to do is use one of the other ones (i'd have to sit down and think hard to figure out exactly which one) to do the merge and then fill the `na` values with the appropriate systems or break down or whatever.

**Comment by gabriel-happle** [comment]
OK. got it. I'll take that issue.

**Comment by gabriel-happle** [comment]
my PR #1188 was deleted without merging. @daren-thomas should I close this? Or can I assign you?

**Comment by JIMENOFONSECA** [comment]
@gabriel-happle let's check again a new solution
